File: app/src/main/java/com/enhance/enhancedemo/network/ApiInterface.java
package com.enhance.enhancedemo.network;
/**
 * Created by riniramac001 on 28/12/16.
 */

import com.enhance.enhancedemo.model.LaunchInfo;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiInterface {

    @GET("launches/")
    Call<List<LaunchInfo>> getSpaceXResponse();

}

File: app/src/main/java/com/enhance/enhancedemo/network/ApiInterface.kt
package com.enhance.enhancedemo.network

import com.enhance.enhancedemo.model.LaunchInfo
import io.reactivex.Observable
import retrofit2.http.GET

/**
 * Created by riniramac001 on 28/12/16.
 */
interface ApiInterface {

    @get:GET("launches/")
    val spaceXResponse: Observable<List<LaunchInfo?>?>?
}